receiver/apachereceiver/testdata/scraper/expected.yaml

```diff
@@ -47,7 +47,7 @@ resourceMetrics:
                     - key: mode
                       value:
                         stringValue: system
-                  timeUnixNano: "1632495518500962000"
+                  timeUnixNano: "500000"
```

Comment:
500,000 seems pretty arbitrary. If we 2x the values it will be easier to visually count:
- `1000000`
- `2000000`